### PR TITLE
Create BasicJava Project

### PR DESCRIPTION
**Pull request by YenPT1**

### CODE REVIEW

File: src/com/gmo/yenpt1/Conditional_Operation.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.gmo.yenpt1;
+
+public class Conditional_Operation {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+int Ten = 10;
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/Conditional_Operation.java]: Format code nhé

File: src/com/gmo/yenpt1/Conditional_Operation.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+int Twenty = 20;
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/Conditional_Operation.java]: Format code nhé

File: src/com/gmo/yenpt1/Conditional_Operation.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+int Thirty = 30;
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/Conditional_Operation.java]: format code nhé

File: src/com/gmo/yenpt1/Conditional_Operation.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+boolean bValue;
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/Conditional_Operation.java]: Format code nhé

File: src/com/gmo/yenpt1/DoubleTestExercise.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package com.gmo.yenpt1;
+
+public class DoubleTestExercise {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		double PI;
+		PI = 3.14159;
+	System.out.println("PI = " + PI);
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/DoubleTestExercise.java]: Format code nhé

File: src/com/gmo/yenpt1/Exercise2.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
+package com.gmo.yenpt1;
+
+public class Exercise2 {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+				
+System.out.println("++++++");
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/Exercise2.java]: Format code nhé

File: src/com/gmo/yenpt1/Exercise2.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+System.out.println("@@@@@");
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/Exercise2.java]: Format code nhé

File: src/com/gmo/yenpt1/IfStatement.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
+package com.gmo.yenpt1;
+
+public class IfStatement {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+String sDay = "Sunday";
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/IfStatement.java]: Format code nhé

File: src/com/gmo/yenpt1/IfStatement.java [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+int iDay = 7;
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/IfStatement.java]: Format code nhé

File: src/com/gmo/yenpt1/IntegerTestExercise.java [hunk 10]
```diff
@@ -0,0 +1,15 @@
+package com.gmo.yenpt1;
+
+public class IntegerTestExercise {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		int carSpeed;
+		carSpeed = 20;
+	System.out.println(" Car is running at the speed of:" + carSpeed);
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/IntegerTestExercise.java]: Format code nhé

File: src/com/gmo/yenpt1/IntegerTestExercise.java [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+		carSpeed = carSpeed + 20;
+	System.out.println("Current speed of the car is: " + carSpeed);
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/IntegerTestExercise.java]: Format code nhé

File: src/com/gmo/yenpt1/RelationalOperator.java [hunk 12]
```diff
@@ -0,0 +1,42 @@
+package com.gmo.yenpt1;
+
+public class RelationalOperator {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+int Ten = 10;
```
Comment (TuyenLMQC) [inline src/com/gmo/yenpt1/RelationalOperator.java]: Format code nhé

File: src/com/gmo/yenpt1/Conditional_Operation.java (hunk 1 above)
Comment (YenPT1) [reply src/com/gmo/yenpt1/Conditional_Operation.java]: Updated

File: src/com/gmo/yenpt1/Conditional_Operation.java (hunk 2 above)
Comment (YenPT1) [reply src/com/gmo/yenpt1/Conditional_Operation.java]: updated

**Pull request merged**